api/utils/helpers.py

```diff
@@ -272,9 +272,6 @@ def validate_token(jwt_token, salt):
     try:
         jwt.decode(jwt_token, key=salt, algorithms=["HS256"])
     except Exception:
-        log.warning(
-            "SUSPICIOUS: JWT token failed to validate with salt %s", redact_value(salt)
-        )
```

Comment:
Do you think it is worth it still logging this so that we know if a JWT token failed? It could be useful and we can do a log.info instead of log.warning and not have the SUSPICIOUS keyword so that it does not alert. What do you think?

Reply:
Makes sense -  I can add that in just in-case we're trying to figure out why we're seeing token validation failures.